daac/variables.tf

```diff
@@ -19,7 +19,6 @@ variable "MATURITY" {
 
 variable "bucket_config" {
   type = map(object({
-    type = string
     oai  = optional(string)
   }))
```

Comment:
Do we want type optional here instead of removed? I guess you can only really get yourself in trouble if you have maturity based types ?

Reply:
Yea, I don't see why you should be changing the bucket type in the overrides

Reply:
As in, I don't think it makes sense to have `type = optional(string)`.

Reply:
I guess this adds an additional constraint that you can't add buckets in the maturity specific config that are not present in the default config.